DEVELOPER_GUIDE.md

```diff
@@ -63,6 +63,11 @@ One easy way to install on mac or linux is to use pip:
 pip install cmake==3.23.3
 ```
 
+On Mac M1 machines, install cmake using:
+```bash
+brew install cmake
```

Comment:
does pip doesn't work on this? also can we add the version of cmake while installing using brew?

Reply:
Pip didn't work for me when I was setting up my new laptop.

Reply:
make sense

Reply:
After further investigation, it seems that homebrew doesn't have a built in way of installing an older version of libraries.  So the only option is to use `brew install cmake` to install the latest version, which is currently 3.27.7